tests/RegressionTests_stampede.intel.log

```diff
@@ -0,0 +1,68 @@
+Mon Oct 19 18:21:16 CDT 2020
+Start Regression test
+
+
+baseline dir = /work/07736/minsukji/stampede2/RT/FV3-MOM6-CICE5/develop-20201012/RT-Baselines_1d_bmwav_ccpp_cmeps
+mediator baseline dir = /work/07736/minsukji/stampede2/RT/FV3-MOM6-CICE5/develop-20201012/RT-Baselines_1d_bmwav_ccpp_cmeps/RESTART
+working dir  = /work/07736/minsukji/stampede2/FV3-MOM6-CICE6/run/stmp3/minsukji/S2S_RT/rt_137288/cpld_fv3_ccpp_384_mom6_cice_cmeps_ww3_1d_bmark_rt
+Checking test 001 cpld_fv3_ccpp_384_mom6_cice_cmeps_ww3_1d_bmark_rt results ....
```

Comment:
It looks to me you do have baseline on stampede2  /work/07736/minsukji/stampede2/RT/FV3-MOM6-CICE5/develop-20201012, is it something you copied from hera/orion? I am curious why the results are different. If baseline is missing, it should give you message "missing baseline", not "NOT OK".

Reply:
This baseline I just copied from Orion.  That is why it is at least comparing.

Reply:
Is it possible that you can make a baseline on Stampede2 (by manually copying the results you already created)? I feel since you may want to make a tag for code delivery for your project, it may be better to have a successful RT test result.